processing/requirements.txt

```diff
@@ -44,7 +44,7 @@ queuelib==1.5.0           # via scrapy
 regex==2020.9.27          # via black, dateparser
 requests==2.24.0          # via -r requirements.in
 s3transfer==0.3.3         # via boto3
-scrapy==2.3.0             # via -r requirements.in
+scrapy==2.4.0             # via -r requirements.in
```

Comment:
**suggestion**: If we use `pip-compile`, all libraries that are required by Scrapy are updated and pinned, so it would be better use it to generate this file.